.circleci/config.yml.in

```diff
@@ -99,6 +99,45 @@ commands:
       - brew_install:
           formulae: libtool
 
+  checkout_and_install:
```

Comment:
now there is `checkout`, `checkout_merge` (which apparently is just `checkout`), and `checkout_and_install`

Should we instead have just `checkout` and `install_torchvision`?